packages/models-library/src/models_library/resource_tracker.py

```diff
@@ -259,7 +259,7 @@ class PricingUnitWithCostUpdate(BaseModel):
     unit_extra_info: UnitExtraInfo
     default: bool
     specific_info: SpecificInfo
-    pricing_unit_cost_update: None | PricingUnitCostUpdate
+    pricing_unit_cost_update: None | PricingUnitCostUpdate = Field(default=None)
```

Comment:
also a change. if these changes you do because of tests. I think the problem lies with the tests and we should not change the interface

Reply:
So, the test has been written afterwards?! What's the purpose of the tests then??